docs/rst/user_manual/wan_configuration.rst

```diff
@@ -101,7 +101,7 @@ TLS
 ===
 
 |eddsrouter| also supports `TLS over TCP <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/tcp/tls.html>`_,
-and its configuration can be set per participant for types Local Discovery Server and WAN. Following is a list of the
+and its configuration can be set per participant for types WAN Discovery Server and WAN. Following is a list of the
```

Comment:
(Note below about CA)

.. note::
Although in principle only required for TLS clients (with peer verification), the CA (Certification- Authority) file may also be provided
for TLS servers when willing to connect them to other participants configured as servers.